### DISCUSSION TITLE
What do I do? Help!

### DISCUSSION THREAD

**Discussion by Myth727** [sunflower-land/sunflower-land#1799] [Q&A]
Haha it's not that bad, just pondering on a problem where I hold too many axes and have no use for them with the foreman beaver. I gathered them just to finish an achievement. Now looking for solutions to my problem and I'm sure anyone else holding the foreman. 

![Screenshot_20221201_145701_MetaMask](https://user-images.githubusercontent.com/96928557/205148305-54e5e3b2-9d11-4158-bf46-4eb73f01a39a.jpg)

**Comment by Galphath** [comment]
Sell them on opensea

**Reply by Myth727** [reply]
Tools are non withdrawable

**Reply by Galphath** [reply]
Ah my bad.  I hope they become tradable soon when LE is open to everybody.

**Comment by future2back** [comment]
I don't understand why the tools are not tradable. I remember some users were using it to withdraw SFL out of the game, maybe thats why its not withdrawable anymore but tradable, there are no arguments for not being. It is interesting that you bring up the subject, because it affects not only foreman but all LVL 3 NFT tools.

**Comment by Myth727** [comment]
I believe I under the reason for no withdrawal on tools. First off there's a different cost when leaving beta over to LE. But just curious if we shall have a vendor or some events that will help me burn some of these axes.

**Reply by future2back** [reply]
> I believe I under the reason for no withdrawal on tools. First off there's a different cost when leaving beta over to LE. But just curious if we shall have a vendor or some events that will help me burn some of these axes.


@Myth727 You can withdraw axes now bro, there are Axes available on opensea
$0.0073 cost on LE
$0,04 selling price on opensea

**Reply by Myth727** [reply]
Not withdrawable on LE, it's no stress for me. I'm just curious if the team plans to put use cases/trades/events where I can use then up.

**Reply by future2back** [reply]
I thought the ones available on OpenSea were from LE.
